Parent/Parent/ParentAppDelegate.swift

```diff
@@ -50,6 +50,17 @@ class ParentAppDelegate: UIResponder, UIApplicationDelegate {
         CacheManager.resetAppIfNecessary()
         if hasFirebase {
             FirebaseApp.configure()
+            let remoteConfig = RemoteConfig.remoteConfig()
+            remoteConfig.activate { error in
+                guard error == nil else {
+                    return
+                }
+                let keys = remoteConfig.allKeys(from: RemoteConfigSource.remote)
+                for key in keys {
+                    UserDefaults.standard.set(remoteConfig.configValue(forKey: key).boolValue, forKey: ExperimentalFeature.settingsKey(forConfigKey: key))
```

Comment:
Ah!, so we don't necessarily have to use `UserDefaults` for this. Seem like a reasonable place to store them though. Makes it easily accessible to RN.

Reply:
👍